daemons/controld/controld_remote_ra.c

```diff
@@ -706,7 +706,8 @@ handle_remote_ra_stop(lrm_state_t * lrm_state, remote_ra_cmd_t * cmd)
     CRM_ASSERT(lrm_state);
     ra_data = lrm_state->remote_ra_data;
 
-    if (ra_data->migrate_status != takeover_complete) {
+    if (!pcmk_is_set(ra_data->status, expect_takeover) &&
+        !pcmk_is_set(ra_data->status, takeover_complete)) {
```

Comment:
only check takeover_complete, we want to hit this regardless of whether expect_takeover is set